lib/db/leveldb_dbinstance_updateschema.go

```diff
@@ -15,12 +16,18 @@ import (
 
 const dbVersion = 4
 
-func (db *Instance) updateSchema() {
+var ErrDatabaseDowngrade = errors.New("the version of the db is more recent than Syncthing and thus not compatible")
```

Comment:
Just a matter of opinion, but I'd prefer something more terse here and have the explanatory stuff all in the log message. Something like `database version from the future` or `unknown database version` maybe?

Reply:
Agree